indexing-service/src/main/java/io/druid/indexing/common/task/RealtimeIndexTask.java

```diff
@@ -107,12 +112,24 @@ private static String makeDatasource(FireDepartment fireDepartment)
   @JsonIgnore
   private volatile QueryRunnerFactoryConglomerate queryRunnerFactoryConglomerate = null;
 
+  @JsonIgnore
+  private volatile IndexMerger indexMerger;
```

Comment:
it appears you can't have final and volatile both, I'm keeping it volatile like other variables just to be consistent for now.